docs/src/ComponentsPage.js

```diff
@@ -733,6 +733,12 @@ const ComponentsPage = React.createClass({
                   <p>Set the <code>header</code> prop to create a structured item, with a heading and a body area.</p>
                   <ReactPlayground codeText={Samples.ListGroupHeader} />
 
+                  <h3><Anchor id="listgroup-with-custom-children">With custom component children</Anchor></h3>
```

Comment:
Does this example add a lot? I think it might be sufficient just to clarify when `componentClass` is needed in the prop documentation for `ListGroup`.